onnxruntime/test/providers/cpu/tensor/unsqueeze_op_test.cc

```diff
@@ -46,7 +46,8 @@ TEST(TensorOpTest, Unsqueeze_3) {
 }
 
 TEST(TensorOpTest, Unsqueeze_Duplicate) {
-  OpTester test("Unsqueeze", -1);  // use latest opset for shape inference errors
+  // latest valid opset for this test is 12. Since opset 13 attribute axes was made an input. 
+  OpTester test("Unsqueeze", 12);
```

Comment:
You can not bump it up. Ditto for everything else that requires implementation change.

Reply:
I reverted the changes to these ops but did not get to the tests. Still working through the validations. I think I removed the WIP label from the PR title too early. 

BTW Specially for this OP... unsqueeze for first updated for bfloat , this change was brought in ort and then it was further updated to move axis as input instead of attribute. Therefore I had to undo the opset bump for this op since another PR is handling the implementation for this per opset13 changes.